### PR TITLE
Max pin Notebook, update/fix markdown in SpecvizExample notebook

### PR DESCRIPTION
**Pull request by rosteen**
Max pin `notebook` to avoid buggy behavior due to incompatibility with Notebook 7. Also fixed some bad text I noticed in a notebook.

### CODE REVIEW

File: notebooks/SpecvizExample.ipynb [hunk 1]
```diff
@@ -54,14 +54,7 @@
...
     "\n",
-    "Note: you probably want to pick one of the three options below depending on preference. Also not the latter two will only work in JupyterLab, not the \"classic\" notebook interface."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This will show Specviz inline in the notebook"
+    "This will show Specviz inline in the notebook. You can pass an integer to the `height` keyword to change the displayed height of the app (the default is 600). You can also pass `loc=\"popout:window\"` to immediately pop out the Jdaviz app into a separate window."
```
Comment (pllim) [inline notebooks/SpecvizExample.ipynb:57]: Seems unrelated but does not hurt.

**Review by pllim — APPROVED** [review]
Unfortunate. Hopefully not long-term.

Will probably need a release soon after.

File: CHANGES.rst [hunk 2]
```diff
@@ -91,6 +91,9 @@ Specviz2d
 Other Changes and Additions
 ---------------------------
 
+- Pinned Jupyter Notebook version to less than 7 to avoid buggy behavior due
```
Comment (pllim) [inline CHANGES.rst:94]: I think you want to move this to 3.6.3 section

**Review by kecnry — APPROVED** [review]
Do we need a follow-up ticket/issue to remove the pin once the issue is sorted out (either on our end or upstream)?

**Comment by pllim** [comment]
> Do we need a follow-up ticket/issue to remove the pin once the issue is sorted out (either on our end or upstream)?

Yes, definitely on GitHub if nowhere else.

Also, there is a conflict here.

**Comment by rosteen** [comment]
The pin is no longer needed after #2420, I'll open a new PR with the notebook markdown update.

**Pull request closed without merging**